### PR TITLE
Update labels to v0.7.0

### PR DESCRIPTION
**Pull request by richardsliu**
**Which issue is resolved by this Pull Request:**
Resolves https://github.com/kubeflow/manifests/issues/524

**Description of your changes:**

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/534)

### CODE REVIEW

File: hack/update-instance-labels.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Replace 'app.kubernetes.io/version: v0.6.x' with 'app.kubernetes.io/version: v0.7.0'
+grep -rl --exclude-dir=hack 'app.kubernetes.io/version: v0.6' ./ | xargs sed -i -E 's/app.kubernetes.io\/version: v0.6(.*)/app.kubernetes.io\/version: v0.7.0/g'
```
Comment (kkasravi) [inline hack/update-instance-labels.sh]: may want to include the same directories for exclusion as hack/gen-test-targets.sh just so we don't inadvertently change something we don't want to

kfdef|gatekeeper|gcp/deployment_manager_configs|aws/infra_configs|doc|tests|hack|plugins

Comment (richardsliu) [reply hack/update-instance-labels.sh]: We do want to update tests, right?

Comment (kkasravi) [reply hack/update-instance-labels.sh]: yep, this does what it needs to then
/lgtm
/approve

**Comment by richardsliu** [comment]
@kkasravi Updated.

**Pull request merged**